.github/workflows/manual-perf.yml

```
@@ -57,6 +57,12 @@ on:
         options:
           - 2ch
           - 8ch
+      extra_args:
+        description: 'Extra arguments for gem5'
+        required: false
+        type: string
+        default: ''
+        description: "Extra arguments for gem5 (e.g., --disable-mgsc). Only works with .py config files."
```

actionlint (1.7.12)

[error] key "description" is duplicated in input settings of workflow_dispatch event. previously defined at line:61,col:9

(syntax-check)